heroku/cli
# fix (pipelines:promote): update promote error handling

Update the `pipelines:promote` command so that it shows the proper error messaging as well as info when running the command. This fix was based on [this ticket](https://heroku.support/958616)

Steps to check:
1. Pull down branch and go to the pipeline's package
2. Run `yarn run mocha test/commands/pipelines/promote.test.ts`
3. Check to make sure tests pass
4. Use an existing pipeline to check that the command properly promotes the corresponding app
